File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    //prompting the user for height input
    do
    {
        height = get_int("height=");
    }
    while (height < 1 || height > 8);
    
    //drawing the left pyramid
    for (int i = 1; i <= height; i++)
    {
        for (int j = 1; j <= (height - i); j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("  ");
        // drawing the right pyramid
        for (int l = 1; l <= i; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int
while(True):
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break
for row in range(1, height+1):
    print(" " * (height - row), end="")
    print("#" * row, end="")
    print("  ", end="")
    print("#" * row, end="")
    print()